pyanaconda/modules/payloads/constants.py

```diff
@@ -33,3 +36,13 @@ class SourceType(Enum):
     LIVE_OS_IMAGE = "LIVE_OS_IMAGE"
     HMC = "HMC"
     CDROM = "CDROM"
+    URL = "URL"
+
+
+@unique
+class URLSourceType(Enum):
```

Comment:
I'm curious - why are the constants and type definition in two places?

Reply:
It's in `core` which is shared for everyone. Modules stuff on the other hand are specific for modules only.

Reply:
Makes sense - thank you!